QEfficient/transformers/pytorch_transforms.py

```diff
@@ -151,5 +154,8 @@ class KVCache(ModuleMapping):
     }
 
     @classmethod
-    def apply(cls, model: nn.Module) -> Tuple[nn.Module, bool]:
-        return super().apply(model)
+    def apply(cls, model: nn.Module):
```

Comment:
The output signature need node be removed.

Reply:
added back